packages/styled-base/types/index.d.ts

```diff
@@ -12,10 +12,9 @@
  * a style of that component.
  */
 
-import { ComponentSelector, Interpolation } from '@emotion/serialize'
 import * as React from 'react'
-
-import { Omit, Overwrapped, PropsOf } from './helper'
+import { ComponentSelector, Interpolation } from '@emotion/serialize'
+import { PropsOf, Omit } from './helper'
```

Comment:
could we rename this Omit to DistributiveOmit? right now it's super easy to miss that this is not the same as builtin one

Reply:
Done